index.html

```diff
@@ -0,0 +1,228 @@
+                    </p>
+                    <h1>2021.10.15(THU) ~ 16(FRI)</h1>
+                    <h3>Webinar on Web Development, Back End, Front End, and
+                        More</h3>
+                </section>
+            </div>
+        </header>
+
+        <section id="main-program">
+            <h2 class="text-center">Main Program</h2>
+            <hr class="short-orange-hr">
+            <ul>
+                <li class="d-flex">
+                    <div class="d-flex course-item">
+                        <img src="images/icon-html.png" alt="html icon">
+                        <h3 class="orange-color">HTML</h3>
+                    </div>
+                    <p class="course-item-description">In this section of the
+                        program we talk about
+                        the
+                        importance and best practices of using HTML semantics in
+                        your front-end website.
+                    </p>
+                </li>
+
+                <li class="d-flex">
+                    <div class="d-flex course-item">
+                        <img src="images/icon-css.png" alt="html icon">
+                        <h3 class="orange-color">CSS</h3>
+                    </div>
+                    <p class="course-item-description">in this section, we talk
+                        about CSS best
+                        practices and
+                        show some examples of that with some topics that are
+                        most useful in CSS like selecting elements, animations,
+                        and more…
+                    </p>
+                </li>
+
+                <li class="d-flex">
+                    <div class="d-flex course-item">
+                        <img src="images/icon-javascript.png" alt="html icon">
+                        <h3 class="orange-color">javaScript</h3>
+                    </div>
+                    <p class="course-item-description">we talk about the
+                        importance of
+                        javascript in front-end
+                        development, javascript best practices, and some
+                        tutorials on javascript in general.
+                    </p>
+                </li>
+
+                <li class="d-flex">
+                    <div class="d-flex course-item">
+                        <img src="images/icon-server.png" alt="html icon">
+                        <h3 class="orange-color">Back End</h3>
+                    </div>
+                    <p class="course-item-description">in this section of the
+                        program we talk
+                        about back-end development in general and in ruby on
+                        rails especially.
+                    </p>
+                </li>
+            </ul>
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+            <button>Join Our 2021 Webinar</button>
+        </section>
```

Comment:
- [ ] The items within the _main program_ section of your design do not have appropriate padding and margins around the items, as well as spacing between the contents of each item. Kindly ensure that there's enough padding around the item's contents.

**Your program section:**
![image](https://user-images.githubusercontent.com/12745474/132545765-01cfcd80-def8-4f69-8dac-60ba494c66ec.png)

**The original program section:**
![image](https://user-images.githubusercontent.com/12745474/132545825-150c3b77-7c05-4a8c-aa95-956269677499.png)